src/Umbraco.Web.BackOffice/Controllers/MemberController.cs

```diff
@@ -286,172 +308,256 @@ public MemberDisplay GetEmpty(string contentTypeAlias = null)
             contentItem.PersistedContent.Email = contentItem.Email;
             contentItem.PersistedContent.Username = contentItem.Username;
 
-            //use the base method to map the rest of the properties
-            base.MapPropertyValuesForPersistence<IMember, MemberSave>(
+            // use the base method to map the rest of the properties
+            MapPropertyValuesForPersistence<IMember, MemberSave>(
                 contentItem,
                 contentItem.PropertyCollectionDto,
-                (save, property) => property.GetValue(), //get prop val
-                (save, property, v) => property.SetValue(v), //set prop val
+                (save, property) => property.GetValue(), // get prop val
+                (save, property, v) => property.SetValue(v), // set prop val
                 null); // member are all invariant
         }
 
-        private IMember CreateMemberData(MemberSave contentItem)
+        /// <summary>
+        /// Create a member from the supplied member content data
+        /// 
+        /// All member password processing and creation is done via the identity manager
+        /// </summary>
+        /// <param name="contentItem">Member content data</param>
+        /// <returns>The identity result of the created member</returns>
+        private async Task CreateMemberAsync(MemberSave contentItem)
         {
-            throw new NotImplementedException("Members have not been migrated to netcore");
-
-            // TODO: all member password processing and creation needs to be done with a new aspnet identity MemberUserManager that hasn't been created yet.
-
-            //var memberType = _memberTypeService.Get(contentItem.ContentTypeAlias);
-            //if (memberType == null)
-            //    throw new InvalidOperationException($"No member type found with alias {contentItem.ContentTypeAlias}");
-            //var member = new Member(contentItem.Name, contentItem.Email, contentItem.Username, memberType, true)
-            //{
-            //    CreatorId = _backofficeSecurityAccessor.BackofficeSecurity.CurrentUser.Id,
-            //    RawPasswordValue = _passwordSecurity.HashPasswordForStorage(contentItem.Password.NewPassword),
-            //    Comments = contentItem.Comments,
-            //    IsApproved = contentItem.IsApproved
-            //};
-
-            //return member;
+            IMemberType memberType = _memberTypeService.Get(contentItem.ContentTypeAlias);
+            if (memberType == null)
+            {
+                throw new InvalidOperationException($"No member type found with alias {contentItem.ContentTypeAlias}");
+            }
+
+            var identityMember = MembersIdentityUser.CreateNew(
+                contentItem.Username,
+                contentItem.Email,
+                memberType.Alias,
+                contentItem.Name);
+
+            // TODO: may not need to add password like this
```

Comment:
@Shazwazza I added this `CreateAsync` overload to include the new password because we are creating it in the back-office - unlike the User creation (which creates the password with ___UIDEMPTYPWORD___ appended). Presume that is right?